### PR TITLE
fix issue 83

### PR DESCRIPTION
**Pull request by JinmingHu-MSFT**
fix https://github.com/Azure/azure-storage-cpplite/issues/83

### CODE REVIEW

**Comment by eyeKill** [comment]
Tested, issue resolved.

File: src/utility.cpp [hunk 1]
```diff
@@ -89,8 +90,23 @@ namespace azure {  namespace storage_lite {
 #else
         pm = std::gmtime(&t);
 #endif
-        size_t s = std::strftime(buf, 30, (format == date_format::iso_8601 ? constants::date_format_iso_8601 : constants::date_format_rfc_1123), pm);
-        return std::string(buf, s);
+        if (format == date_format::iso_8601)
+        {
+            char buf[32];
+            std::strftime(buf, sizeof(buf), constants::date_format_iso_8601, pm);
```
Comment (katmsft) [inline src/utility.cpp:96]: Why iso_8601 does not need locale adjustment?

Comment (JinmingHu-MSFT) [reply src/utility.cpp:96]: talked offline

Comment (JinmingHu-MSFT) [reply src/utility.cpp:96]: because iso_8601 is locale-independent

**Pull request merged**